packages/backend/src/plugins/search.ts

```diff
@@ -26,7 +28,7 @@ export default async function createPlugin(
     timeout: { minutes: 15 },
     // A 3 second delay gives the backend server a chance to initialize before
     // any collators are executed, which may attempt requests against the API.
```

Comment:
Please remove this comment

Reply:
Sorry, forgot about that. Fixed in the latest commit.